Add combustion and fuels exam (Jan 2019) (#660)
Gp2mv3/Syntheses: src/q7/combu-MECA2160/exam/2019/Janvier/All/combu-MECA2160-exam-2019-Janvier-All.tex

    \item Consider the following system of mono-molecular reactions
        \begin{align*}
            \cee{ A + 2M &<=>[k_f][k_r] A^* + 2M } \\
            \cee{ A^* &->[k_p] Pr }
        \end{align*}
        $A^*$ being a radical and $M$ being a ``catalyst''. By employing the steady-state approximation, derive the equation for the concentration of the species $\left[A\right]$ at high and low pressures. (25 \%)
    \item Identify and discuss (without mathematical equations) the processes in the $\mathrm{H}_2/\mathrm{O}_2$ system that result in the first and second explosion limits. (25 \%)
    \item With regard to the turbulent premixed flames, give the definitions of the Damkohler number and the turbulent Reynolds number, \emph{Da} and \emph{Re}$_T$ respectively. Give the definition and a brief description of the three different flame regimes and represent them on the \emph{Re}$_T$ -- \emph{Da} plane. Identify the flame regime to which the eddy-breakup model is applicable and explain the idea upon which this model is based. (25 \%)
\end{enumerate}

> 

Reviewer: ```suggestion

\nosolution

```

\section{Exercise 1}
Consider the following reaction mechanism for the production of HBr from H$_2$ and Br:
\begin{align*}
    \cee{M + Br_2 &-> Br + Br + M} \\
    \cee{M + Br + Br &-> Br_2 + M} \\
    \cee{Br + H_2 &-> HBr + H} \\
    \cee{HBr + H &-> Br + H_2}
\end{align*}
\begin{enumerate}
    \item For each reaction, identify the type of elementary reaction (e.g. unimolecular etc.) and indicate its role in the chain mechanism (e.g. chain initiation etc.) (10 \%)
    \item Write the complete expressions for teh reaction rates of \ce{Br_2} and \ce{Br}, i.e. $\frac{dc_{\mathrm{Br}_2}}{dt}$ and $\frac{dc_{\mathrm{Br}}}{dt}$. (20 \%)
    \item Apply the steady-state approximation to derive an expression for the concentration of the H radical. (20 \%)
\end{enumerate}

> 

Reviewer: ```suggestion

\nosolution

```

\section{Exercise 2}
During the combustion of a piece of wood that contains 25\% humidity, we measured the following concentrations in the combusiton products: $\left[\ce{CO2}\right]'$ = 0.12 and $\left[\ce{O_2}\right]'$ = 0.085. The formulation of wood is \ce{CH_{1.44}O_{0.66}}. Design the Ostwald diagram for wood and identify this particular combustion on it. How does humidity effect the measured concentrations of species in the combustion products? (50 \%)

> 

Reviewer: ```suggestion

\nosolution
```

\documentclass[en]{../../../../../../eplexam}
\usepackage[version=3]{mhchem}

\hypertitle{Combustion and fuels}{7}{MECA}{2160}{2019}{Janvier}{All}
{Adrien Couplet}
{Miltiadis Papalexandris}

> The exam mark is distributed as follows: 50 \% theory, 35 \% exercises and 15 \% laboratory report.

Reviewer 2: ```suggestion
The exam marks are distributed as follows: 50 \% theory, 35 \% exercises and 15 \% laboratory report.
```

\section{Theory}
\begin{enumerate}
    \item Give the definition of the Lower Heating Value (LHV) of a fuel at constant pressure. Derive the expression for the temperature of the combustion products of an adiabatic combustion, $T_{as}$, as a function of the LHV of the fuel and the temperature of the reactants. (25 \%)
    \item Consider the following system of mono-molecular reactions
        \begin{align*}
            \cee{ A + 2M &<=>[k_f][k_r] A^* + 2M } \\
            \cee{ A^* &->[k_p] Pr }
        \end{align*}
        $A^*$ being a radical and $M$ being a ``catalyst''. By employing the steady-state approximation, derive the equation for the concentration of the species $\left[A\right]$ at high and low pressures. (25 \%)
    \item Identify and discuss (without mathematical equations) the processes in the $\mathrm{H}_2/\mathrm{O}_2$ system that result in the first and second explosion limits. (25 \%)

>     \item With regard to the turbulent premixed flames, give the definitions of the Damkohler number and the turbulent Reynolds number, \emph{Da} and \emph{Re}$_T$ respectively. Give the definition and a brief description of the three different flame regimes and represent them on the \emph{Re}$_T$ -- \emph{Da} plane. Identify the flame regime to which the eddy-breakup model is applicable and explain the idea upon which this model is based. (25 \%)

Reviewer 2: Techniquement, les nombres de Damkohler et de Reynolds devraient être typesets avec `\mathrm`, mais voilà...

```suggestion
    \item With regard to the turbulent premixed flames, give the definitions of the Damkohler number and the turbulent Reynolds number, $\mathrm{Da}$ and $\mathrm{Re}_\textnormal{T}$ respectively. Give the definition and a brief description of the three different flame regimes and represent them on the $\mathrm{Re}_\textnormal{T}$ -- $\mathrm{Da}$ plane. Identify the flame regime to which the eddy-breakup model is applicable and explain the idea upon which this model is based. (25 \%)
```

\end{enumerate}

\nosolution

\section{Exercise 1}

> Consider the following reaction mechanism for the production of HBr from H$_2$ and Br:

Reviewer 2: ```suggestion
Consider the following reaction mechanism for the production of \ce{HBr} from \ce{H2} and \ce{Br}:
```

\begin{align*}
    \cee{M + Br_2 &-> Br + Br + M} \\
    \cee{M + Br + Br &-> Br_2 + M} \\
    \cee{Br + H_2 &-> HBr + H} \\
    \cee{HBr + H &-> Br + H_2}
\end{align*}
\begin{enumerate}
    \item For each reaction, identify the type of elementary reaction (e.g. unimolecular etc.) and indicate its role in the chain mechanism (e.g. chain initiation etc.) (10 \%)

>     \item Write the complete expressions for teh reaction rates of \ce{Br_2} and \ce{Br}, i.e. $\frac{dc_{\mathrm{Br}_2}}{dt}$ and $\frac{dc_{\mathrm{Br}}}{dt}$. (20 \%)

Reviewer 2: ```suggestion
    \item Write the complete expressions for the reaction rates of \ce{Br2} and \ce{Br}, i.e. $\frac{\dif c_{\ce{Br2}}}{\dif t}$ and $\frac{\dif c_{\ce{Br}}}{\dif t}$. (20 \%)
```

>     \item Apply the steady-state approximation to derive an expression for the concentration of the H radical. (20 \%)

Reviewer 2: ```suggestion
    \item Apply the steady-state approximation to derive an expression for the concentration of the \ce{H} radical. (20 \%)
```

\end{enumerate}

\nosolution

\section{Exercise 2}

> During the combustion of a piece of wood that contains 25\% humidity, we measured the following concentrations in the combusiton products: $\left[\ce{CO2}\right]'$ = 0.12 and $\left[\ce{O_2}\right]'$ = 0.085. The formulation of wood is \ce{CH_{1.44}O_{0.66}}. Design the Ostwald diagram for wood and identify this particular combustion on it. How does humidity effect the measured concentrations of species in the combustion products? (50 \%)

Reviewer 3: ```suggestion
During the combustion of a piece of wood that contains 25\% humidity, we measured the following concentrations in the combustion products: $\left[\ce{CO2}\right]'$ = 0.12 and $\left[\ce{O_2}\right]'$ = 0.085. The formulation of wood is \ce{CH_{1.44}O_{0.66}}. Design the Ostwald diagram for wood and identify this particular combustion on it. How does humidity effect the measured concentrations of species in the combustion products? (50 \%)
```

> During the combustion of a piece of wood that contains 25\% humidity, we measured the following concentrations in the combusiton products: $\left[\ce{CO2}\right]'$ = 0.12 and $\left[\ce{O_2}\right]'$ = 0.085. The formulation of wood is \ce{CH_{1.44}O_{0.66}}. Design the Ostwald diagram for wood and identify this particular combustion on it. How does humidity effect the measured concentrations of species in the combustion products? (50 \%)

Reviewer 2: ```suggestion
During the combustion of a piece of wood that contains 25 \% humidity, we measured the following concentrations in the combustion products: $\left[\ce{CO2}\right]' = 0.12$ and $\left[\ce{O_2}\right]' = 0.085$. The formulation of wood is \ce{CH_{1.44}O_{0.66}}. Design the Ostwald diagram for wood and identify this particular combustion on it. How does humidity effect the measured concentrations of species in the combustion products? (50 \%)
```
